TF_Service_and_Tooling_Unity/Assets/TouchFree/ServiceUI/Scripts/DiagnosticAPI.cs

```diff
@@ -0,0 +1,227 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using UnityEngine;
+using WebSocketSharp;
+
+public class DiagnosticAPI : IDisposable
+{
+    private static string uri = "ws://127.0.0.1:1024/";
+
+    public enum Status { Closed, Connecting, Connected, Expired }
+    private Status status = Status.Expired;
+    private WebSocket webSocket = null;
+
+    public static event Action<ImageMaskData> OnGetMaskingResponse;
+    public static event Action<bool> OnMaskingVersionCheck;
+
+    public uint connectedDeviceID;
+    public bool maskingAllowed = false;
+
+    const string minimumMaskingAPIVerison = "2.0.0";
+
+    ConcurrentQueue<string> newMessages = new ConcurrentQueue<string>();
+
+    public DiagnosticAPI(MonoBehaviour _creatorMonobehaviour)
+    {
+        Debug.Log("DiagnosticAPI constructor... ");
```

Comment:
Do we need this alert in generally / for testing purposes? If the latter (or not needed) we should probably knock it out before merge

Reply:
Yeah very true, there's probably a few of these logs we can remove